pkg/sync/v14/data/ClusterRole.rbac.authorization.k8s.io/customer-admin-project.yaml

```diff
@@ -41,3 +41,12 @@ rules:
   - delete
   - patch
   - update
+- apiGroups:
+  - ""
```

Comment:
add the line ` - image.openshift.io` here

Reply:
Will update